zerver/models.py

```diff
@@ -214,8 +215,8 @@ def remote_user_to_email(remote_user):
 
 class RealmEmoji(models.Model):
     realm = models.ForeignKey(Realm)
-    name = models.TextField()
-    img_url = models.TextField()
+    name = models.TextField(validators=[MinLengthValidator(1), RegexValidator(regex=r'^[0-9a-zA-Z_\-\.]*$')])
+    img_url = models.URLField()
```

Comment:
What happens to existing instances of a model when you change the type of a field from TextField to URLField?  Seems like this may involve a messy migration.